website/models.py

```diff
@@ -1,9 +1,40 @@
-from .__init__ import db
+from . import db
 from flask_login import UserMixin
 from sqlalchemy.sql import func
+from django.db import models
+from django.contrib.auth.models import User
+from PIL import Image
+
+
+class Note(db.Model):
```

Comment:
are you using Note in profile related stuff? Because I removed it the other day, if its relevant to your codes then lmk

Reply:
Eh tak tak relevant you could just remove it